virttest/utils_net.py

```diff
@@ -4145,3 +4145,21 @@ def get_netkvm_param_value(vm, param):
         return value
     finally:
         session.close()
+
+
+def get_host_ipv4_addr():
```

Comment:
https://github.com/avocado-framework/avocado-vt/blob/5c83fe5eafe748f771adaa406879c02d24c90faf/virttest/utils_net.py#L3388-L3407

Reply:
@PaulYuuu  Thanks for your reminder, but get_host_ip_address does not meet our requirement for now. There is a param netdst(virbr0) but no hot_ip_addr, in this case, it will pick up IP address on virbr0.

Reply:
```python
def get_host_ip_address(params=None, ifname=None, ip_ver="ipv4", linklocal=False):
    if params:
        host_ip = params.get('host_ip_addr', None)
        if host_ip:
            logging.debug("Use IP address at config %s=%s", 'host_ip_addr', host_ip)
            return host_ip
        net_dev = params.get("netdst")
    else:
        net_dev = ifname
    if not net_dev:
        net_dev = get_host_default_gateway(iface_name=True)
```
I think this change can help you, and it is compatible with the previous way.

Reply:
@PaulYuuu  Thanks for the so detailed resolution, updated.

Reply:
It's nothing. `utils_net.py` looks good to me, other code needs someone help to review.